core/services/vrf/v2/listener_v2.go

```diff
@@ -72,29 +73,8 @@ const (
 	// backoffFactor is the factor by which to increase the delay each time a request fails.
 	backoffFactor = 1.3
 
-	V2ReservedLinkQuery = `SELECT SUM(CAST(meta->>'MaxLink' AS NUMERIC(78, 0)))
-		FROM evm.txes
-		WHERE meta->>'MaxLink' IS NOT NULL
-		AND evm_chain_id = $1
-		AND CAST(meta->>'SubId' AS NUMERIC) = $2
-		AND state IN ('unconfirmed', 'unstarted', 'in_progress')
-		GROUP BY meta->>'SubId'`
-
-	V2PlusReservedLinkQuery = `SELECT SUM(CAST(meta->>'MaxLink' AS NUMERIC(78, 0)))
-		FROM evm.txes
-		WHERE meta->>'MaxLink' IS NOT NULL
-		AND evm_chain_id = $1
-		AND CAST(meta->>'GlobalSubId' AS NUMERIC) = $2
-		AND state IN ('unconfirmed', 'unstarted', 'in_progress')
-		GROUP BY meta->>'GlobalSubId'`
-
-	V2PlusReservedEthQuery = `SELECT SUM(CAST(meta->>'MaxEth' AS NUMERIC(78, 0)))
-		FROM evm.txes
-		WHERE meta->>'MaxEth' IS NOT NULL
-		AND evm_chain_id = $1
-		AND CAST(meta->>'GlobalSubId' AS NUMERIC) = $2
-		AND state IN ('unconfirmed', 'unstarted', 'in_progress')
-		GROUP BY meta->>'GlobalSubId'`
+	SubId       = "SubId"
+	GlobalSubId = "GlobalSubId"
```

Comment:
Maybe better to prefix these with `TxMetaField` to unambiguously indicate that these are referring to TxMeta fields.

```suggestion
	TxMetaFieldSubId       = "SubId"
	TxMetaGlobalSubId      = "GlobalSubId"
```

Reply:
BTW, do these need to be public constants?

Reply:
Good call, they do not. Updated them accordingly.